drivers/wifi/nrf700x/zephyr/src/zephyr_wifi_mgmt.c

```diff
@@ -584,6 +584,8 @@ int wifi_nrf_set_twt(const struct device *dev,
 		if (twt_params->teardown.teardown_all) {
 			if (vif_ctx_zep->neg_twt_flow_id == 0xFF) {
 				LOG_ERR("Invalid negotiated TWT flow id\n");
+				twt_params->fail_reason =
```

Comment:
This seems unrelated to the commit log, please remove or make it a separate commit.

Reply:
Added a separate commit.